app/assets/stylesheets/_base.scss

```diff
@@ -936,8 +936,18 @@ body {
 
     .content {
       margin-left: 20px;
+
+      a {
+        color: $dark_purple;
```

Comment:
To make the services a different color of the rest of the texst, but not too different. This color already existed in the defined variables.